source/_UIAConstants.py

```diff
@@ -0,0 +1,62 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2021 NV Access Limited
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+
+import enum
+
+
+class FillType(enum.IntEnum):
+	none = 0
+	color = 1
+	gradient = 2
+	picture = 3
+	pattern = 4
+
+
+FillTypeLabels = {
+	# Translators: a style of fill type (to color the inside of a control or text)
+	FillType.none: pgettext("UIAHandler.FillType", "none"),
+	# Translators: a style of fill type (to color the inside of a control or text)
+	FillType.color: pgettext("UIAHandler.FillType", "color"),
+	# Translators: a style of fill type (to color the inside of a control or text)
+	FillType.gradient: pgettext("UIAHandler.FillType", "gradient"),
+	# Translators: a style of fill type (to color the inside of a control or text)
+	FillType.picture: pgettext("UIAHandler.FillType", "picture"),
+	# Translators: a style of fill type (to color the inside of a control or text)
+	FillType.pattern: pgettext("UIAHandler.FillType", "pattern"),
+}
+
+
+# Some newer UIA constants that could be missing
+class UIAutomationType(enum.IntEnum):
+	Int = 1
+	Bool = 2
+	String = 3
+	Double = 4
+	Point = 5
+	Rect = 6
+	Element = 7
+	Array = 8
+	Out = 9
+	IntArray = 10
+	BoolArray = 11
+	StringArray = 12
+	DoubleArray = 13
+	PointArray = 14
+	RectArray = 15
+	ElementArray = 16
+	OutInt = 17

```

Comment:
```suggestion
	OUT_INT = 17
```

Reply:
My goal was to keep the constants as close to what is in UIAutomationCore.idl. They start with an uppercase letter, and contain no underscores. Similar to other constants in UI automation.
I can convert all these to uppercase, but I'm not so sure on the underscore, or what the rule there is. But I want to be really sure this is necessary as it will be a big change which needs to be checked.
@feerrenrut  your thoughts?

Reply:
I'd like to have a consistent style, and we have previously chosen uppercase for enums. I understand the desire to reflect UI Automation, but I think it is also good to make our code clearly ours. I should be able to update this fairly quickly with an IDE, I'll push a change shortly.